teuthology/task/install/__init__.py

```diff
@@ -441,6 +441,18 @@ def task(ctx, config):
              - url: "http://location.repo"
                name: "ceph_repo"
 
+    Add repo before trying to install any package, the shaman
+    related tasks will be ignored.
+
+    Note: This only valid for SUSE based distro, other systems
+    are welcome to support.
```

Comment:
Note: This is valid for SUSE-based distros only: other distros are welcome to support.